hack/run-conformance.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+echo "Running sonobuoy conformance e2e test suite"
+export KUBECONFIG=$(./kubevirtci kubeconfig)
+export TENANT_CLUSTER_NAME=${TENANT_CLUSTER_NAME:-kvcluster}
+export TENANT_CLUSTER_NAMESPACE=${TENANT_CLUSTER_NAMESPACE:-kvcluster}
+
+sonobuoy_release="https://github.com/vmware-tanzu/sonobuoy/releases/download/v0.56.8/sonobuoy_0.56.8_linux_amd64.tar.gz"
+sonobuoy_result_tarball="results.tar.gz"
+sonobuoy_plugin="e2e"
+sonobuoy_pod_runner="${TENANT_CLUSTER_NAME}-sonobuoy-runner-$(echo $RANDOM | sha1sum | head -c 4)"
+sonobuoy_conformance_dir="/tmp/sonobuoy_conformance"
+sonobuoy_status="sonobuoy_status.json"
+
+cat <<EOF | ./kubevirtci kubectl create -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: ${sonobuoy_pod_runner}
+  namespace: ${TENANT_CLUSTER_NAMESPACE}
+spec:
+  containers:
+  - name: sonobuoy
+    image: registry.access.redhat.com/ubi8/ubi:8.0
+    env:
+    - name: KUBECONFIG
+      value: /etc/kubernetes/kubeconfig/value
+    - name: E2E_SKIP
+      value: ""
+    - name: SONOBUOY_PLUGIN
+      value: $sonobuoy_plugin
+    command:
+    - /bin/bash
+    - -c
+    - |
+      dnf install -y jq
+      curl -L $sonobuoy_release -o sonobuoy.tar.gz && tar -xzf sonobuoy.tar.gz
+      chmod +x sonobuoy
+      SONOBUOY_EXTRA_ARGS=${SONOBUOY_EXTRA_ARGS:-""}
+      ./sonobuoy run --plugin \$SONOBUOY_PLUGIN --wait $SONOBUOY_EXTRA_ARGS
+      ./sonobuoy retrieve -f $sonobuoy_result_tarball
+      ./sonobuoy status --json > $sonobuoy_status
+      sleep 3600
```

Comment:
is this sleep just needed to ensure we can pull the artifacts from the pod after the tests complete?

Reply:
Correct